test-case/check-audio-equalizer.sh

```diff
@@ -110,7 +123,7 @@ do
     rate=$(func_pipeline_parse_value "$idx" rate)
     fmt=$(func_pipeline_parse_value "$idx" fmt)
     type=$(func_pipeline_parse_value "$idx" type)
-    eq_support=$(func_pipeline_parse_value "$idx" eq)
+    eq_support=($(func_pipeline_parse_value "$idx" eq))
```

Comment:
Maybe something like this? UNTESTED
```shell
 func_pipeline_parse_value "$idx" eq | read -d ' ' -a eq_support
```

Pipelines drop errors by default but in this case `func_pipeline_parse_value` would very likely have already failed earlier.

Reply:
It doesn't work but I don't know why. Just `func_pipeline_parse_value "$idx" eq` seems to return e.g.

`eqiir.21.1 eqfir.21.1` or `eqiir.41.0`